lib/bitreserve/api/contact.rb

```diff
@@ -0,0 +1,34 @@
+module Bitreserve
+  module API
+    module Contact
+      def all_contacts
+        request_data = RequestData.new(
+          Endpoints::USER_CONTACTS,
+          Entities::Contact,
+          authorization_header
+        )
+        Request.perform_with_objects(:get, request_data)
+      end
+
+      def find_contact(id: nil)
+        request_data = RequestData.new(
+          Endpoints::USER_CONTACTS + "/#{id}",
+          Entities::Contact,
+          authorization_header
+        )
+        Request.perform_with_object(:get, request_data)
+      end
+
+      def create_contact(**args)
```

Comment:
Right. I was thinking that we could use required key arguments, but that would mean dropping support for 2.0, which I'm not that comfortable with, so let's go with the double splat.